python/meterpreter/meterpreter.py

```diff
@@ -630,6 +640,48 @@ def seek(self, offset, whence=os.SEEK_SET):
     def tell(self):
         raise NotImplementedError()
 
+#@export
+class MeterpreterLog(MeterpreterChannel):
```

Comment:
It might make sense to make this logging async, so any written debug data can immediately be flushed to the listening msfconsole - as an alternative to polling for available data:

https://github.com/rapid7/metasploit-payloads/blob/2b659882d0261d06494fddbfe43a5614f7dc3c83/python/meterpreter/meterpreter.py#L1384-L1399

Reply:
We can implement this approach after we've got MVP windows C implementation confirmed as working